pkg/server/plugin/datastore/datastore.go

```diff
@@ -24,9 +24,9 @@ type DataStore interface {
 
 	// Entries
 	CountRegistrationEntries(context.Context) (int32, error)
-	CreateRegistrationEntry(context.Context, *CreateRegistrationEntryRequest) (*CreateRegistrationEntryResponse, error)
-	DeleteRegistrationEntry(context.Context, *DeleteRegistrationEntryRequest) (*DeleteRegistrationEntryResponse, error)
-	FetchRegistrationEntry(context.Context, *FetchRegistrationEntryRequest) (*FetchRegistrationEntryResponse, error)
+	CreateRegistrationEntry(context.Context, *common.RegistrationEntry) (*common.RegistrationEntry, error)
+	DeleteRegistrationEntry(ctx context.Context, entryID string) (*common.RegistrationEntry, error)
```

Comment:
The interface didn't have variable names before because it was just a codegen'd interface off of the gRPC client and the real documentation was in the proto. However, now that we're moving away from that, I think we should name the variables. It is not only self-documenting but makes it easier to write the function comment.

Reply:
Hmm. On the previous PR #2231 @azdagron asked for the variable names